opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -1016,28 +1024,44 @@ namespace Opm
         const auto& glo = schedule.glo(report_step_idx);
         if (!glo.has_well(well_name)) {
             const std::string msg = fmt::format(
-                "GLIFT WTEST: Well {} : Gas Lift not activated: "
+                "GLIFT WTEST: Well {} : Gas lift not activated: "
                 "WLIFTOPT is probably missing. Skipping.", well_name);
             deferred_logger.info(msg);
             return;
         }
         const auto& gl_well = glo.well(well_name);
-        auto& max_alq_optional = gl_well.max_rate();
-        Scalar max_alq;
-        if (max_alq_optional) {
-            max_alq = *max_alq_optional;
+
+        // Use gas lift optimization to get ALQ for well test
+        std::unique_ptr<GasLiftSingleWell> glift =
+            initializeGliftWellTest_<GasLiftSingleWell>(simulator,
+                                                        well_state,
+                                                        group_state,
+                                                        phase_usage,
+                                                        ecl_well_map,
+                                                        deferred_logger);
+        auto [max_alq, success] = glift->wellTestALQ();
+        std::string msg;
```

Comment:
Is this the max? Isn't this the min alq that allows the well to open?

Reply:
Indeed, forgot to change the old variable name. Fixed now.